src/wasm/wasm-binary.cpp

```diff
@@ -1694,19 +1732,20 @@ void WasmBinaryBuilder::readImports() {
       }
       case ExternalKind::Table: {
         Name name(std::string("timport$") + std::to_string(tableCounter++));
-        wasm.table.module = module;
-        wasm.table.base = base;
-        wasm.table.name = name;
+        auto table = std::make_unique<Table>();
+        table->module = module;
+        table->base = base;
+        table->name = name;
```

Comment:
We should probably add a `makeTable` method to wasm-builder.h to cover this kind of usage.